src/_help/enhance-documentation-content/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Enhance your documentation content
+---
+
+- TOC
+{:toc}
+
+API documentation deserve a bit more writing. It can be useful to add an introduction or a quick tutorial. You might want to provide context, additional reading or understanding materials.
```

Comment:
```suggestion
API documentation deserves a bit more writing than a plain technical description. It can be useful to add an introduction or a quick tutorial. You might want to provide context, additional reading, or understanding materials.
```